src/Assertion/StringTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Omnipay\PayPal\Assertion;
+
+trait StringTrait
```

Comment:
I would suggest to remove this strait and just use docblocks for type-hinting until strict typing becomes available in Omnipay.

Reply:
yes I will, update coming up, please stop the reviewing :stuck_out_tongue: